includes/class-wc-stripe-apple-pay-registration.php

```diff
@@ -228,6 +228,7 @@ public function process_apple_pay_verification() {
 
 		} catch ( Exception $e ) {
 			$this->stripe_settings['apple_pay_domain_set'] = 'no';
+			$this->apple_pay_domain_set                    = false;
```

Comment:
Looks like this is needed to show the error if it fails after having succeeded earlier – sort of an edge case but good catch nonetheless!